lara-typescript/src/section-authoring/api/mock-api-provider.ts

```diff
@@ -137,23 +161,86 @@ const deletePageItem: APIPageItemDeleteF = (args: {pageId: PageId, pageItemId: I
 
 const getAllEmbeddables = () => {
   const allEmbeddables: ISectionItemType[] = [
-    {id: "1", name: "Carousel", useCount: 1, dateAdded: 1630440496},
-    {id: "2", name: "CODAP", useCount: 5, dateAdded: 1630440497},
-    {id: "3", name: "Drag & Drop", useCount: 5, dateAdded: 1630440498},
-    {id: "4", name: "Fill in the Blank", useCount: 8, dateAdded: 1630440495},
-    {id: "5", name: "iFrame Interactive", useCount: 200, dateAdded: 1630440494, isQuickAddItem: true},
-    {id: "6", name: "Multiple Choice", useCount: 300, dateAdded: 1630440493},
-    {id: "7", name: "Open Response", useCount: 400, dateAdded: 1630440492, isQuickAddItem: true},
-    {id: "8", name: "SageModeler", useCount: 3, dateAdded: 1630440499},
-    {id: "9", name: "Teacher Edition Window Shade", useCount: 4, dateAdded: 1630440490},
-    {id: "10", name: "Text Box", useCount: 500, dateAdded: 1630440491, isQuickAddItem: true}
+    {
+      id: "1",
+      name: "Carousel",
+      type: "ManagedInteractive",
+      useCount: 1,
+      dateAdded: 1630440496
+    },
+    {
+      id: "2",
+      name: "CODAP",
+      type: "ManagedInteractive",
+      useCount: 5,
+      dateAdded: 1630440497
+    },
+    {
+      id: "3",
+      name: "Drag & Drop",
+      type: "ManagedInteractive",
+      useCount: 5,
+      dateAdded: 1630440498
+    },
+    {
+      id: "4",
+      name: "Fill in the Blank",
+      type: "ManagedInteractive",
+      useCount: 8,
+      dateAdded: 1630440495
+    },
+    {
+      id: "5",
+      name: "iFrame Interactive",
+      type: "MwInteractive",
+      useCount: 200,
+      dateAdded: 1630440494,
+      isQuickAddItem: true
+    },
+    {
+      id: "6",
+      name: "Multiple Choice",
+      type: "ManagedInteractive",
+      useCount: 300,
+      dateAdded: 1630440493
+    },
+    {
+      id: "7",
+      name: "Open Response",
+      type: "ManagedInteractive",
+      useCount: 400,
+      dateAdded: 1630440492,
+      isQuickAddItem: true
+    },
+    {
+      id: "8",
+      name: "SageModeler",
+      type: "ManagedInteractive",
+      useCount: 3,
+      dateAdded: 1630440499
+    },
+    {
+      id: "9",
+      name: "Teacher Edition Window Shade",
+      type: "ManagedInteractive",
+      useCount: 4,
+      dateAdded: 1630440490
+    },
+    {
+      id: "10",
+      name: "Text Block",
+      type: "Embeddable::Xhtml",
+      useCount: 500,
+      dateAdded: 1630440491,
+      isQuickAddItem: true
+    }
   ];
   return Promise.resolve({allEmbeddables});
 };
 
 export const API: IAuthoringAPIProvider = {
   getPages, getPage, createPage, deletePage,
   createSection, updateSections, updateSection,
-  createPageItem, deletePageItem,
-  getAllEmbeddables
+  createPageItem, updatePageItem, deletePageItem,
+  getAllEmbeddables, pathToTinyMCE: null, pathToTinyMCECSS: undefined
```

Comment:
I wonder what we could do for the mock-api-provider as far as loading TinyMCE, is there a CDN URL we could use?  We don't have to do anything now if its hard, just wondering if there is something easy to do here.

Reply:
Good idea. I added a CDN URL in the latest commit. I'm not sure if that may need to be changed frequently, but I doubt it. And I think we'll eventually drop support for TinyMCE anyway, so it may not be an issue for long.